src/publish-nuget.target

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why are you duplicating this from the buildtools package? Why not just use the one from build tools just like the buildtools repo itself does.